netfields/fields.py

```diff
@@ -14,7 +14,10 @@
     MACAddress8FormField
 )
 from netfields.mac import mac_unix_common, mac_eui64
-from netfields.psycopg2_types import Inet, Macaddr, Macaddr8
+try:
+    from netfields.psycopg2_types import Inet, Macaddr, Macaddr8
+except ImportError:
+    from netfields.psycopg3_types import Inet, Macaddr, Macaddr8
```

Comment:
This fails on Django 4.2 when both psycopg2 and 3 are available. I know of an environment where this will be the case :-)

Since Django is using it, we should try to make sure we are loading the right one. Perhaps importing from whatever module Django loads it from.

Reply:
Fixed in 5963486. There's a variable `is_psycopg3` exposed by the Django/Postgres module which we can hook into.